### PR TITLE
Jk/develop/feature/add modal

### PR DESCRIPTION
**Pull request by neu5**

### CODE REVIEW

File: src/js/main.es6 [hunk 1]
```diff
@@ -0,0 +1,77 @@
+$(document).ready(function () {
```
Comment (CezaryDanielNowak) [inline src/js/main.es6]: `$(function () {` would be just enough

File: src/js/main.es6 [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  let modal = function Modal() {
```
Comment (CezaryDanielNowak) [inline src/js/main.es6:2]: why not just `function Modal` ? And why uppercase, is it a constructor function?

File: src/js/main.es6 [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+      $overlay: $overlay
+    };
+  }();
+
+  modal.$closeBtn.on('click', modal.close);
+  modal.$overlay.on('click', function(ev) {
+    if (ev.target.id === 'overlay') {
+      modal.close();
+    }
+  });
```
Comment (CezaryDanielNowak) [inline src/js/main.es6:63]: I'd recommend to move modal to separate file/module

Comment (neu5) [inline src/js/main.es6:63]: :+1:

File: src/js/main.es6 (hunk 2 above)
Comment (neu5) [inline src/js/main.es6:2]: it is constructor function but as a singleton, I want just one instance of it :D

File: src/js/main.es6 (hunk 3 above)
Comment (neu5) [inline src/js/main.es6:63]: some another time

**Pull request merged**